client/web/src/site-admin/routes.tsx

```diff
@@ -131,7 +132,19 @@ export const otherSiteAdminRoutes: readonly SiteAdminAreaRoute[] = [
         render: () => <AnalyticsNotebooksPage />,
     },
     {
-        path: '/configuration',
+        path: '/configuration/',
+        render: () => (
+            <RedirectRoute
+                getRedirectURL={({ location }) =>
+                    `${location.pathname.replace('/configuration', '/configuration/basic')}${location.search}${
+                        location.hash
+                    }`
+                }
+            />
+        ),
+    },
+    {
+        path: '/configuration/:tab',
```

Comment:
I guess, this was added to make both root `/configuration` and new `/configuration/:tab` paths render correctly. However, this changes original behavior even when `setup-checklist` ff is off. I think, we can make this simpler by just using optional route parameters:
```suggestion
    {
        path: '/configuration/:tab?',
```

Reply:
FYI, I've tested locally, and this works, so gonna push this change.

Reply:
Nice, thanks. I was thinking about this as well, but for some reason thought it would be easier to just redirect.